Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -247,6 +247,25 @@ class RUMViewScopeTests: XCTestCase {
 
     // MARK: - User Action Tracking
 
+    func test_finalizeActiveUserActionsOnStopView() throws {
```

Comment:
Let's keep the same behaviour-testing convention as used in all RUM scope tests:
```suggestion
    func testGivenActiveUserAction_whenViewIsStopped_itSendsUserActionEvent() throws {
```
It is way easier to understand the tests this way, as they map 1:1 to product requirements.